xml/System.Net.Sockets/TcpClient.xml

```diff
@@ -1672,7 +1672,7 @@ The `GetStream` method returns a <xref:System.Net.Sockets.NetworkStream> that yo
       <Docs>
         <summary>Gets or sets a value that disables a delay when send or receive buffers are not full.</summary>
         <value>
-          <see langword="true" /> if the delay is disabled, otherwise <see langword="false" />. The default value is <see langword="false" />.</value>
+          <see langword="true" /> if the delay is disabled, otherwise, <see langword="false" />. The default value is <see langword="false" />.</value>
```

Comment:
```suggestion
          <see langword="true" /> if the delay is disabled; otherwise, <see langword="false" />. The default value is <see langword="false" />.</value>
```